build/grunt.js

```diff
@@ -14,6 +14,10 @@ module.exports = function(grunt) {
     patch: verParts[2]
   };
 
+  version.majorMinor = `${version.major}.${version.minor}`;
```

Comment:
This part should get merged in regardless (I can open a new PR if we don't want the rest of this PR) because it could actually break things since `version.majorMinor` was being used before it was defined.